src/plugins/intel_cpu/tests/functional/shared_tests_instances/subgraph_tests/gather_weights_decompression.cpp

```diff
@@ -45,4 +45,20 @@ INSTANTIATE_TEST_SUITE_P(smoke_GatherCompressedWeights_basic,
                                             ::testing::ValuesIn(per_tensor_scale)),
                          GatherWeightsDecompression::get_test_case_name);
 
+// fp16/bf16 constant + convert(16bit to f32) + gather case
+TEST_P(GatherWeightsDecompressionWithoutScale, CompareWithRefs) {
+    SKIP_IF_CURRENT_TEST_IS_DISABLED()
+    run();
+}
+const std::vector<ov::element::Type> weights_precisions_wo_scale = {ov::element::f16, ov::element::bf16};
+const std::vector<ov::element::Type> output_precisions_wo_scale = {ov::element::f32};
```

Comment:
Does it even make sense to configure `output_precision`? I mean, isn't it always f32 in real cases?

- If it is f16 and weights are f16, then there is no decompression convert at all. 
- Another case: f16 output precision and bf16 weights precision. But I am not sure that this is a combination that we should test?

@dmitry-gorokhov @chenhu-wang what do you think?

Reply:
maybe it's needed functionally such as fp16 IR with bf16 infer_precision and vice versa, the convert elimination could be done by other pass. added.